### ISSUE TITLE
Title override for main page

### ISSUE THREAD

**Issue by geoffreymcgill** [retypeapp/retype#350]
### Discussed in https://github.com/retypeapp/retype/discussions/346

<div type='discussions-op-text'>

<sup>Originally posted by **DannyBen** June 26, 2022</sup>
I am looking for an ability to completely override the title of the main page, for search engine visibility purposes.

Right now, the main page is handled like any other page, and receiving the title suffix as defined in the config's `meta.title`:

```yaml
meta:
  title: " | Bashly - Bash CLI Framework"
```

I was looking at [retype's `title` page configuration](https://retype.com/configuration/page/#title), but it is obviously not intended for this.

Can this be done? If not, can this be considered as a feature?

Ideally, I think this should be definable at the page level:

```yaml
# page front matter
title_override: "My search-engine-friendly title"  # which will appear without the global suffix
---
```</div>

**Comment by geoffreymcgill** [comment]
Hi @DannyBen. Good question and an interesting scenario. Maybe the same `meta.title` config in the page metadata would be a good option. 

```yml
meta:
  title: "Custom meta title here which overrides "
```

Where setting `meta.title` completely overrides the default functionality of the generated `<title>` value. Whatever you set at the page config level would be the exact value set in the `<title>` meta tag for that generated page.

Let me know if you like the `meta.title` config name. This config option would be easy for us to add.

**Comment by DannyBen** [comment]
Yes, sounds perfectly logical and intuitive.

I am guessing the current `title` in the page-level config still needs to stay and retain its current meaning?

**Comment by geoffreymcgill** [comment]
This feature has been implemented and will be available in the upcoming Retype v3.0 release.
